roles/openldap/tasks/install-configure.yml

```diff
@@ -0,0 +1,80 @@
+- name: install the openldap packages
+  apt: name={{ item }} state=installed
+  with_items: "{{ openldap.ldap_pkgs }}"
+
+- name: create directory for ldif files
+  file: path={{ openldap.config_dir }}
+        state=directory
+        owner=root
+        group=root
+        mode=0400
+
+- name: copy templates
+  template: src={{ item }}
+            dest={{ openldap.config_dir }}/{{ item | basename }}
+  with_fileglob: ../templates/install_configure/*
+
+- name: set log level
+  shell: ldapmodify -Q -Y EXTERNAL -H ldapi:/// -f {{ openldap.config_dir }}/logging.ldif
```

Comment:
This is idempotent, it will overwrite the existing log level with a new or same one if log levels don't change between plays.